modules/cluster-logging-updating-prereqs.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-upgrading.adoc
+
+[id="cluster-logging-updating-prereqs_{context}"]
+= Prerequisites for updating
+
+Updating cluster logging to 4.5 also upgrades your cluster to use Elasticsearch 6.  Before updating cluster logging to 4.5, you must perform the following tasks to prepare your cluster to work with the new version of Elasticsearch.
+
+.Prerequisites
+
+* Update cluster logging from 4.5 to 4.6.
+
+* Make sure the cluster logging status is healthy:
++
+** All Pods are `ready`.
+** Elasticsearch cluster is healthy.
+
+.Procedure
+
+As a user of logging, I want to search my application logs and filter them by label before and after upgrade of the log stack
```

Comment:
Not sure why we are adding use cases here or even documenting them generally.  IMO this should be removed